Formula/gofabric8.rb

```diff
@@ -0,0 +1,36 @@
+class Gofabric8 < Formula
+  desc "CLI for fabric8 running on Kubernetes or OpenShift"
+  homepage "https://github.com/fabric8io/gofabric8/"
+  url "https://github.com/fabric8io/gofabric8/archive/v0.4.69.tar.gz"
+  sha256 "21a34018367d8aeb9446be23747ff918770d0c8e827e7d3e0b7a57542eb61274"
+
+  depends_on "go" => :build
+
+  GOVENDOR_PATH = "src/github.com/fabric8io/gofabric8".freeze
+
+  def install
+    ENV["GOPATH"] = buildpath
+    files = Dir["*"]
+    mkdir_p buildpath/GOVENDOR_PATH
```

Comment:
`mkdir do "src/github.com/fabric8io/gofabric8"` will create it and `cd` to it temporarily